src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>NPSP Getting Started Page section items</description>
+    <fields>
+        <fullName>Description_Label__c</fullName>
+        <description>The custom label api name used in the description on a section item on the Getting Started page</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The custom label api name for the Description Label</inlineHelpText>
```

Comment:
@smarovich Do you think the inlineHelpText should the description for all of these? There is a mix of some that match and some that don't. I wasn't sure if you had explicitly asked to have the description different than the help text. In some cases there is more information/detail in the description and in others there is more help text.
cc @fparodi-salesforce @dcaballeroSf

Reply:
@force2b I have seen it both ways in the past. Sometimes, the Description may include more information for Admins. But, we've also just used the same text for both description and inlinehelptext in the past too. I'm ok with both approaches. If you're worried about consistency, we can make the values match.

Reply:
Thanks @smarovich. In this case, I think it's useful that the InlineHelpText have at least as much detail as Description. The two new objects/fields in this PR are NPSP-System objects that - ideally - only _we_ will ever update as new features/menu options are added to the Getting Started Adoption pages. However, the teams that add those new features by creating new records in these objects need as much guidance as possible on what values to put into fields and how those values are used.

Reply:
Thanks for the info, @force2b. Makes perfect sense to me. I'll make some suggested edits to the Descriptions/Help Text and mention you (so you can make sure they're providing enough information for other teams that might use these fields).